src/vs/workbench/browser/workbench.contribution.ts

```diff
@@ -789,7 +789,24 @@ const registry = Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Con
 					localize('confirmBeforeCloseWeb', "Controls whether to show a confirmation dialog before closing the browser tab or window. Note that even if enabled, browsers may still decide to close a tab or window without confirmation and that this setting is only a hint that may not work in all cases.") :
 					localize('confirmBeforeClose', "Controls whether to show a confirmation dialog before closing a window or quitting the application."),
 				'scope': ConfigurationScope.APPLICATION
-			}
+			},
+			'window.unresponsive': {
```

Comment:
Are we expecting users to configure this setting after all? Can we find a good default and just go with that?

Reply:
Just leaving it an option, depending on the issue maybe users want to collect stacks at different frequency and intervals.

Reply:
Lets add that when we feel the need for it and just use defaults for now.